### PR TITLE
Beta release (v0.0.1)

### PR DESCRIPTION
**Pull request by hulien22**
Modifying README, reconfiguring some of the build steps, adding build files, updating plugin.json.

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-31`

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-31`

**Comment by hulien22** [comment]
Deciding to remove the dist folder from this pr, we will create a new `release` branch that will contain our releases as well as this folder

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-31`

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-31`

File: src/plugin.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "type": "datasource",
   "name": "Cognite Data Platform",
-  "id": "cognitedata-platform-datasource",
+  "id": "cognitedata-datasource",
```
Comment (martincognite) [inline src/plugin.json]: proposal; `cognite-datasource` instead of `cognitedata-datasource`

**Comment by cognite-cicd** [comment]
[pr-bot]
Run this build with `docker run --rm -p 3000:3000 cognite/grafana-cdp-dev:pr-31`

**Pull request merged**